tests/phpunit/Unit/Cache/ElementsCacheBuilderTest.php

```diff
@@ -45,6 +45,8 @@ public function testCanConstruct() {
 
 	public function testGetTermsForSingleTermWithDefinitionOnNonCachedResult() {
 
+		$this->markTestSkipped( 'Needs to be fixed with the new version of SG' );
```

Comment:
Why do we have to wait for the next release before the tests can be fixed? If there are specific dependencies we should name them.